homeassistant/components/efergy/sensor.py

```diff
@@ -156,8 +176,8 @@ def update(self):
                 for sensor in response.json():
                     if self.sid == sensor["sid"]:
                         measurement = next(iter(sensor["data"][0].values()))
-                        self._state = measurement
+                        self._attr_state = measurement
             else:
-                self._state = None
+                self._attr_state = None
```

Comment:
@frenck If coverage for this line is required, I'm not sure how. It's probably something simple. 😄